bindings/hnsw_bindings.cpp

```diff
@@ -125,10 +138,14 @@ template <typename dist_t, typename data_t = float> class Index {
   }
 
   void init_new_index(const size_t maxElements, const size_t M,
-                      const size_t efConstruction, const size_t random_seed) {
+                      const size_t efConstruction, const size_t random_seed,
+                      py::object using_pq) {
```

Comment:
Wouldnt it be better to pass a const reference instead of needing to pass a copy?